src/app/Plans/components/PlansTable.tsx

```diff
@@ -239,17 +240,24 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
           <Pagination {...paginationProps} widgetId="plans-table-pagination-top" />
         </LevelItem>
       </Level>
-      <Table
-        aria-label="Migration Plans table"
-        className="plans-table"
-        cells={columns}
-        rows={rows}
-        sortBy={sortBy}
-        onSort={onSort}
-      >
-        <TableHeader />
-        <TableBody />
-      </Table>
+      {filteredItems.length > 0 ? (
+        <Table
+          aria-label="Migration Plans table"
+          className="plans-table"
+          cells={columns}
+          rows={rows}
+          sortBy={sortBy}
+          onSort={onSort}
+        >
+          <TableHeader />
+          <TableBody />
+        </Table>
+      ) : (
+        <TableEmptyState
+          titleText="No migration plans found"
+          bodyText="No results match your filter. Go back and make a different selection."
```

Comment:
Nit-pick, but the "Go back" instruction only makes sense in the wizard, I think we can leave this as just "No results match your filter."